mir_eval/separation.py

```diff
@@ -52,9 +52,9 @@ def validate(reference_sources, estimated_sources):
 
     Parameters
     ----------
-    reference_sources : np.ndarray, shape=(nsrc, nsampl)
+    reference_sources : np.ndarray, shape=(nsrc1, nsampl1)
```

Comment:
Yeah, my logic for that sounds hollow to me. I agree, that should be the convention for the docstring. I'll change that back.